NEWS.md

```diff
@@ -1,10 +1,11 @@
 # naomi 2.9.28
 
-* Make duckdb an optional dependency
+* Suppress "some observations have no neighbours" and "neighbour object has 2 sub-graphs" warnings from `spdep` v1.3.6. We expect this warning for some countries and it will make tests and output noisy to leave on.
```

Comment:
Can you add the link to the spdep vignette describing the package change? (for future me when I forget again)